### DISCUSSION TITLE
Supervisor non presente

### DISCUSSION THREAD

**Discussion by effe-7** [indomus/forum#2677] [HUB: Home Assistant]
Ho installato homeassistant prima con il dogle poi con virtual box essenzialmente perché vorrei accedere al supervisor per provare integrazione con i dispositivi sonoff/ewelink (non modificati) ma non riesco a vedere.
Eppure in informazioni di sistenma Home Assistant Supervisor risulta attivo.

Qualche suggerimento?
Grazie

**Comment by snakuzzo** [comment]
Scusa ma a cosa vorresti accedere? Per installare gli addon non c'è alcuna voce supervisor su cui cliccare.
Vai su impostazioni > componenti aggiuntivi.
Per le varie integrazioni da GUI invece, ti basta andare su integrazioni > dispositivi e servizi.
Per integrazioni custom vai sui progetti delle integrazioni oppure usa HACS

**Comment by effe-7** [comment]
Sulla guida di welink riporta la voce supervisor come si vede in screenshot 

![image](https://github.com/indomus/forum/assets/83612865/5675acdb-383b-4c05-b9b7-0d6b8c07ea55)

**Reply by snakuzzo** [reply]
Non esiste più quella voce...quella guida non sarà aggiornata. Comunque ti ho già detto prima cosa fare

**Reply by effe-7** [reply]
Grazie mi ero incaponito sulle vecchie indicazioni ewelink

**Reply by KiLiMaToS** [reply]
Ciao, ma quindi dove hai installato l’addon di eWeLink? Ci sei riuscito? Grazie
